FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDeviceAgnosticTests.m

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (c) 2013, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <FBSnapshotTestCase/FBSnapshotTestCase.h>
+
+@interface FBSnapshotTestCaseDeviceAgnosticTests : FBSnapshotTestCase
+
+@end
+
+@implementation FBSnapshotTestCaseDeviceAgnosticTests
+
+- (void)setUp
+{
+    [super setUp];
+    self.deviceAgnostic = YES;
+    self.recordMode = NO;
+}
+
+- (void)testViewSnapshot
+{
+    UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    redView.backgroundColor = [UIColor redColor];
+    FBSnapshotVerifyView(redView, nil);
```

Comment:
hmm I just cloned your PR, [UIApplication sharedApplication] is nil at this point. I believe mainly because of the project doesn't have a host application.

![screen shot 2015-12-16 at 11 23 43 am](https://cloud.githubusercontent.com/assets/185995/11851423/7b583e44-a3e7-11e5-85d6-b1dfc7226b60.png)